public/gameInfo.php

```diff
@@ -734,12 +734,12 @@ function submitSetRequest(user, gameID) {
                     echo "<div>Relations</div>";
                     echo "<table><tbody>";
                     if (!empty($relatedGames)) {
-                        echo "<tr><td>";
+                        echo "<tr>";
```

Comment:
Removing `<td>` here is technically not valid HTML, well, at least not very clean.
Did you remove it for layout reasons?

Reply:
I removed the table altogether and cleaned up the markup a bit

Reply:
I was having issues with the width expanding beyond the leftcontainer with the `<td>` still being used. Looks like just removing the table works fine. Thanks.